dev/ci/ci-coquelicot.sh

```diff
@@ -3,8 +3,6 @@
 ci_dir="$(dirname "$0")"
 . "${ci_dir}/ci-common.sh"
 
-install_ssreflect
```

Comment:
This is wrong, coquelicot depends on ssr. Not sure what makes you think it's already installed.

Reply:
So, should I add `install_foo` in all jobs that depend on `foo`?

Reply:
No, ssreflect is treated specially (because it's part of mathcomp but we don't want all mathcomp here)

Reply:
I'd like to remove this "exception", now coquelicot depends on mathcomp proper.
if you want, we can split mathcomp in 2.